openstack/sharedfilesystems/v2/snapshots/requests.go

```diff
@@ -5,6 +5,56 @@ import (
 	"github.com/gophercloud/gophercloud/pagination"
 )
 
+// CreateOptsBuilder allows extensions to add additional parameters to the
+// Create request.
+type CreateOptsBuilder interface {
+	ToSnapshotCreateMap() (map[string]interface{}, error)
+}
+
+// CreateOpts contains the options for create a Snapshot. This object is
+// passed to snapshots.Create(). For more information about these parameters,
+// please refer to the Snapshot object, or the shared file systems API v2
+// documentation
+type CreateOpts struct {
+	// The UUID of the share from which to create a snapshot
+	ShareID string `json:"share_id" required:"true"`
+	// Indicates whether snapshot creation must be attempted when a share’s status is not
+	// available. Set to true to force snapshot creation when the share is busy performing
+	// other operations. Default is false.
+	Force *bool `json:"force,omitempty"`
```

Comment:
Where is this defined in the Manila API code? I see that it is defined in the API docs, but I am unable to see this defined in the service-side code you linked to.

Sometimes the API docs are out of date - maybe this is no longer a valid field?

Reply:
JFYI: https://bugs.launchpad.net/manila/+bug/1811336

Reply:
Thank you for looking into this. I'll be able to do another review later on today.